wheels/social-auth-core/meta.yml

```diff
@@ -0,0 +1,8 @@
+---
+
+type: wheel
+name: social-auth-core
+version: 3.0.0+gx0
```

Comment:
Is this for Galaxy? 19.01 requires 1.5.0, while `dev` requires 3.1.0 .

Reply:
Yes for Galaxy. It seems the 3.1.0 was never released on github: https://github.com/python-social-auth/social-core/releases

I will use 3.1.0+gx0 then.